Aquality.WinAppDriver/src/Aquality.WinAppDriver/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using OpenQA.Selenium;
+using System.Linq;
+using System.Reflection;
+
+namespace Aquality.WinAppDriver.Extensions
+{
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Returns name of the value from <see cref="Keys"/>, readable in the log.
+        /// </summary>
+        /// <param name="key">Keyboard key to define.</param>
+        /// <returns>Readable value.</returns>
+        public static string GetLoggableValueForKeyboardKey(this string key)
```

Comment:
Why it is public? It looks like helper for some library actions, but not as public method that should be available in client code.

Reply:
don't see any reason why should we hide the functionality that could be used by the customer.
I can imagine situations when I want to log this value but I cannot.
Seems that I should rework this keys to public enum...